decred/pyproject.toml

```diff
@@ -30,11 +30,30 @@ pynacl = "^1.3.0"
 websocket_client = "^0.57.0"
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.2"
-pytest-cov = "^2.8.1"
-pyflakes = "^2.1.1"
+pytest = "^5.3"
+pytest-cov = "^2.8"
+flake8 = "^3.7"
 black = "^19.10b0"
+isort = "^4.3"
+
+[tool.isort]
+atomic = "true"
+combine_as_imports = "true"
+combine_star = "true"
+filter_files = "true"
+force_grid_wrap = 0
+force_sort_within_sections = "true"
+include_trailing_comma = "true"
```

Comment:
I don't quite get this one. What's it supposed to do? Docs say

> Will set isort to automatically add a trailing comma to the end of from imports.

But I don't see that anywhere.

Reply:
Here's an [example](https://github.com/pypa/pip/commit/353a13ddff2d4c1e66598e2fb68985498d9095ef) of its application.

And here's a [discussion](https://github.com/timothycrosley/isort/issues/694) about isort interoperability with black where its use is suggested.

(As an aside it looks like quoting `true` values in TOML files is not needed, I have to try that.)